python/mlx/utils.py

```diff
@@ -155,7 +155,7 @@ def tree_flatten(tree, prefix="", is_leaf=None):
     return [(prefix[1:], tree)]
 
 
-def tree_unflatten(tree):
+def tree_unflatten(tree: Any) -> Any:
```

Comment:
The type here for tree does not match the type in the doc string. I would use the type in the docstring which is more specific.

Reply:
Thank you for letting me know, I just added the change